Doc/tutorial/datastructures.rst

```diff
@@ -510,7 +511,7 @@ value associated with that key is forgotten.  It is an error to extract a value
 using a non-existent key.
 
 Performing ``list(d.keys())`` on a dictionary returns a list of all the keys
-used in the dictionary, in arbitrary order (if you want it sorted, just use
+used in the dictionary, in insertion order (if you want it sorted, just use
```

Comment:
maybe add example how to  "permanently" sort a dict ? like:

```python
xxx = dict(sorted(xxx.items(), key=operator.itemgetter(0))) 
```

Reply:
It’s make sense, thanks.

Reply:
I don't think so.  This is tutorial.  We should focus about very important part.

Previously, "unordered" was important for new users to avoid pitfall.
On the other hand, "preserving insertion order" is not so important.